### PR TITLE
Admin Panel UI does not reflect user state

### PR DESCRIPTION
**Pull request by Gavinok**
**Description**
The admin panel currently allows the admin user to change the role of a user and enable/disable them. However, the user's current role is not shown, and the "enable" button is actually an enable/disable toggle, producing confusing results when the admin presses "enable" when the user is already enabled (this will disable the user). The buttons themselves work in the backend, but the UI should reflect the present state

**Reproduction steps**
Steps to reproduce the behaviour:
1. Log in as admin
2. Go to the admin panel
3. Note that the role dropdown defaults to "Open this select menu"
4. Note that all enable toggles say "enable user"

**Expected behaviour**
* "Open this select menu" should default to the user's present role
* "Enable user" should say "disable user" if the user is already enabled

**Additional context / Screenshots**
image

**Story points**
3

### CODE REVIEW

**Comment by Gavinok** [comment]
Not yet ready to merge since due to the spring backend needing to be adjusted

**Comment by Gavinok** [comment]
For reference this is how it looks.

**Comment by Gavinok** [comment]
This task as completed in approximately 5 hours

File: src/components/pages/AdminPanel.js [hunk 1]
```diff
@@ -1,93 +1,116 @@
...
-    </SimplePageLayout>
-  );
-};
+	// Possible Roles the text to display for it
+	const roles = {
+		"ROLE_STAFF": "STAFF",
+		"ROLE_USER": "USER"
+	}
+	const listOfUsersHTML = () => {
+		console.log(listOfUsers)
```
Comment (ACHultman) [inline src/components/pages/AdminPanel.js:60]: extra console log here and on line 63

File: src/components/pages/Login.js [hunk 2]
```diff
@@ -7,11 +7,13 @@ import CenteredHeader from "../UI/atoms/CenteredHeader.js";
...
-  const { setUserInfo } = useContext(UserContext);
-  const [auth, setAuth] = useState(false);
-  const [message, setMessage] = useState(false);
+const Login = (props) => {
 
+	const { context } = props;
+	const {user, setUserInfo,logout} = useContext(UserContext);
+	const [auth, setAuth] = useState(false);
+	const [message, setMessage] = useState(false);
+	console.log("Userinfo", user);
```
Comment (ACHultman) [inline src/components/pages/Login.js:16]: extra log

File: src/service/AdminPanel/AdminPanel.js [hunk 3]
```diff
@@ -8,6 +8,10 @@ export function enableAccount(username,token){
 	return doGet("http://localhost:8000/api/venus/admin/enableuser?enable=true&username="+username, token)	
 }
 
+export function disableAccount(username,token){
+	return doGet("http://localhost:8000/api/venus/admin/enableuser?enable=false&username="+username, token)	
+}
+
 export function changeAccountRole(username, role, token){
 	return doGet("http://localhost:8000/api/venus/admin/changerole?username="+username+"&role="+role, token)
```
Comment (ACHultman) [inline src/service/AdminPanel/AdminPanel.js:16]: I do wish these were POST but out of scope for this PR

**Review by ACHultman — APPROVED** [review]

**Pull request merged**